specification/dartLangSpec.tex

```diff
@@ -11870,14 +11916,12 @@ \subsubsection{Getter Access and Method Extraction}
 Consider an unconditional property extraction $i$
 (\ref{propertyExtraction})
 of the form \code{$e$.\id}.
-It is a compile-time error if \id{} is a member of class \code{Object}
-and $e$ is either a prefix namespace (\ref{imports})
-or a type literal.
+It is a compile-time error if \id{} is the name of
+a member of the built-in class \code{Object}
```

Comment:
-> an instance member

Object may get static members, they are not important here.